src/pages/LandingPage.jsx

```diff
@@ -0,0 +1,11 @@
+import Home from '../pages/Home';
+
+const LandingPage = () => {
+  return (
+    <div>
+      <Home />
+    </div>
+  );
```

Comment:
Do not wrap an element in extra div which is not doing anything, also the component is just a wrapper of another component, move the code in `Home` into `LandingPage` component